test/Williamson2ShallowWaterRosenbrockTests.jl

```diff
@@ -0,0 +1,86 @@
+module Williamson2ShallowWaterRosenbrockTests
+
+using FillArrays
+using Test
+using WriteVTK
+using Gridap
+using GridapGeosciences
+
+# Solves the steady state Williamson2 test case for the shallow water equations on a sphere
+# of physical radius 6371220m. Involves a modified coriolis term that exactly balances
+# the potential gradient term to achieve a steady state
+# reference:
+# D. L. Williamson, J. B. Drake, J. J.HackRüdiger Jakob, P. N.Swarztrauber, (1992)
+# J Comp. Phys. 102 211-224
+
+# Constants of the Williamson2 test case
+const α  = π/4.0              # deviation of the coriolis term from zonal forcing
```

Comment:
The same comment as in https://github.com/gridapapps/GridapGeosciences.jl/pull/18/files#r711833021 applies here for the `Williamsom2` initial conditions.

Reply:
done